bin/ocaml/doc.ml

```diff
@@ -44,7 +44,7 @@ let term =
         | Darwin -> Some ("open", [ "-u" ])
         | Linux -> Some ("xdg-open", [])
         | Windows -> None
-        | Other -> None
+        | Other | FreeBSD | NetBSD | OpenBSD -> None
```

Comment:
FTR BSD platforms can have open or xdg-open but it's probably not worth adding support for it in this PR.